pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "netexec"
-version = "1.1.0"
+version = "0.0.0" # we set this to 0.0.0 so that the version is dynamically set by poetry-dynamic-versioning

```

Comment:
How does it work then? How do we increase the version?

Reply:
It's via git tags